--- a/system/vl.c
+++ b/system/vl.c
@@ -3066,9 +3066,10 @@
             fake_argv[fake_argc++] = strdup("-drive");
             char *escaped_hdd_path = strdup_double_commas(hdd_path);
-            fake_argv[fake_argc++] = g_strdup_printf("index=0,media=disk,file=%s%s",
+            fake_argv[fake_argc++] = g_strdup_printf("index=0,media=disk,file=%s%s,cache=writeback",
                 escaped_hdd_path,
                 strlen(escaped_hdd_path) > 0 ? ",locked=on" : "");
             free(escaped_hdd_path);
         }
     }
